### ISSUE TITLE
Rubinius 2.n LLVM installation instructions on macOS are out of date

### ISSUE THREAD

**Issue by chrisseaton** [rbenv/ruby-build#1079]
When I try to `rbenv install rbx-2.11` on macOS, I'm told to `brew install llvm@3.5`, but this doesn't work any more `No available formula with the name "llvm@3.5"`.

Why do I want to install such an old version of Rubinius? Because the JIT was removed from newer versions and the new one isn't in place yet.

What do I do instead of `brew install llvm@3.5`.

cc @brixen

**Comment by hsbt** [comment]
I understood your concern. But instruction of llvm-3.5.x installation is not ruby-build responsibility scope, I think.

**Comment by chrisseaton** [comment]
But this code and the error message is part of ruby-build isn't it?

https://github.com/rbenv/ruby-build/blob/master/bin/ruby-build#L862-L893

```bash
require_llvm() {
  local llvm_version="$1"
  if is_mac -ge 1010; then
    if [[ "$RUBY_CONFIGURE_OPTS" != *--llvm-* ]]; then
      case "$llvm_version" in
      3.2 )
        package_option ruby configure --prebuilt-name="llvm-3.2-x86_64-apple-darwin13.tar.bz2"
        ;;
      3.[56] )
        local llvm_config="$(locate_llvm "$llvm_version")"
        if [ -n "$llvm_config" ]; then
          package_option ruby configure --llvm-config="$llvm_config"
        else
          local homebrew_package="llvm@$llvm_version"
          { echo
            colorize 1 "ERROR"
            echo ": Rubinius will not be able to compile using Apple's LLVM-based "
            echo "build tools on OS X. You will need to install LLVM $llvm_version first."
            echo
            colorize 1 "TO FIX THE PROBLEM"
            echo ": Install Homebrew's llvm package with this"
            echo -n "command: "
            colorize 4 "brew install $homebrew_package"
            echo
          } >&3
          return 1
        fi
        ;;
      esac
    fi
  fi
}
```

It's not code that is in the Rubinius tarball.

`ruby-build` is the only place that this message can be fixed.

**Comment by hsbt** [comment]
ok, I removed/modified this code.

Your original concern caused by Homebrew. So, [llvm@3.5](https://github.com/Homebrew/homebrew-core/blob/master/Formula/llvm@3.5.rb) is not available now, Only 4.0-3.7.

**Comment by hsbt** [comment]
Duplicates of #862
